exercises/all-your-base/canonical-data.json

```diff
@@ -1,23 +1,14 @@
 {
   "exercise": "all-your-base",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "comments": [
-    "It's up to each track do decide:",
+    "This canonical data makes the following choices:",
+    "1. Zero is always represented in outputs as [0] instead of [].",
+    "2. In no other instances are leading zeroes present in any outputs.",
+    "3. Leading zeroes are accepted in inputs.",
```

Comment:
it should be relatively obvious that the opposite choice is "rejected", so a clause, "rather than being rejected" was not added.